lib/traject/config/folio_config.rb

```diff
@@ -2600,6 +2600,10 @@ def accumulate_summary_struct_fields(matching_fields, tag, label, marc, accumula
   }
 end
 
+to_field 'bound_with_parent_item_ids_ssim' do |record, accumulator|
+  accumulator << record.index_items.filter_map { |holding| holding.bound_with_holding&.dig('boundWith', 'item', 'id') }
```

Comment:
```suggestion
  accumulator.concat record.index_items.filter_map { |holding| holding.bound_with_holding&.dig('boundWith', 'item', 'id') }
```

Otherwise we'll get `[['A', 'B', 'C']]`, right?

Should this have a test?